keras/backend/tensorflow_backend.py

```diff
@@ -389,6 +389,7 @@ def variable(value, dtype=None, name=None, constraint=None):
         array([[ 1.,  2.],
                [ 3.,  4.]])
     ```
+    {{np_implementation}}
```

Comment:
There is no numpy equivalent of `variable` (well, we have one, but only for testing purposes). I believe we should not display the numpy version in the docs because it's going to confuse users.

Reply:
I got it. Thanks to advice.